packages/opc-timeline/README.md

```diff
@@ -0,0 +1,171 @@
+# opc-timeline Component 👋
+
+![Version](https://img.shields.io/badge/version-0.0.1-blue.svg?cacheSeconds=2592000)
+![Build Status](https://travis-ci.org/dwyl/esta.svg?branch=master)
+[![Maintenance](https://img.shields.io/badge/Maintained%3F-yes-green.svg)](https://github.com/1-Platform/op-components/graphs/commit-activity)
+
+A web component based on Lit Element to show a timeline for a specific range
+---
+## Usage
+
+### Install opc-timeline
+
+```sh
+npm install --save @one-platform/opc-timeline 
+```
+
+### For VanillaJS
+- Import component
+```js
+import '@one-platform/opc-timeline';
+```
+- Add component in html
+```html
+<opc-timeline id="timeline" current-step-index="2" variant="compact">
+    <div slot="timeline-details">
+        <h2>Timeline Details</h2>
+        The timeline component details go right here, it uses a slot named <strong>timeline-details</strong>. Happy coding :)
+    </div>
+</opc-timeline>
+```
+- Using the steps attribute we can add an Array of strings which would be the steps for the timeline component
+```js
+document.querySelector('#timeline').steps = [ 'Scheduled', 'Loaned', 'Extended', 'Completed'];
+```
+
+### For Angular
+- In your app.module include the `CUSTOM_ELEMENTS_SCHEMA` and import the component
+```js
+import { NgModule, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+import '@one-platform/opc-timeline';
+
+@NgModule({
+  declarations: [
+    AppComponent,
+  ],
+  imports: [
+    BrowserModule,
+  ],
+  schemas: [CUSTOM_ELEMENTS_SCHEMA],
+  providers: [],
+  bootstrap: [AppComponent]
+})
+export class AppModule { }
+```
+- Add component in any component html template
+```html
+<opc-timeline id="timeline" current-step-index="2" variant="compact">
+    <div slot="timeline-details">
+        <h2>Timeline Details</h2>
+        The timeline component details go right here, it uses a slot named <strong>timeline-details</strong>. Happy coding :)
+    </div>
+</opc-timeline>
+```
+
+### For React
+- Import the component in App.js
+```js
+import '@one-platform/opc-timeline';
+```
+
+- Add component in any component html render
+```html
+<opc-timeline id="timeline" current-step-index="2" variant="compact">
+    <div slot="timeline-details">
+        <h2>Timeline Details</h2>
+        The timeline component details go right here, it uses a slot named <strong>timeline-details</strong>. Happy coding :)
+    </div>
+</opc-timeline>
+```
+---
+## Slots
+- There are three (2 + 1) optional slots in the opc-timeline as shown below namely
```

Comment:
this looks a bit confusing to me. can we just say that there are 3 optional slots? also in the bullets can u write in this format. `<slot-name>: description`
for eg: `"start-label":  The start label on the left side of the timeline`  if u want to provide a example then provide that in sub bullet

Reply:
Yes, changed the structure